src/spryker/harness.yml

```
@@ -89,7 +89,7 @@ attributes:
       php-fpm: "= 'my127/spryker:' ~ @('php.version') ~ '-fpm-' ~ (@('php.version') >= 7.4 ? 'buster' : 'stretch')"
   elasticsearch:
     image: elasticsearch
-    tag: '6.8.22'
+    tag: "= @('spryker.demoshop-version') <= '202009.0'? '6.8.23' : '7.16.3'"
```

```suggestion
    tag: "= version_compare(@('spryker.demoshop-version'), '202009.0', '<=') ? '6.8.23' : '7.16.3'"
```

version_compare is less likely to have issues